CMakeLists.txt

```diff
@@ -49,15 +65,9 @@ if (WITH_WS_LIBHTTPC OR WITH_WS_WSLAY OR WITH_WS_CPPREST)
     set(HAVE_DEFAULT_RT_TRANSPORT_FACTORY ON)
 endif()
 
-if (ANDROID)
-    SET(ANDROID_USE_LEGACY_TOOLCHAIN_FILE FALSE)
```

Comment:
Android upstream abandoned "new" toolchain and legacy toolchain is actually only one maintained going forward: https://developer.android.com/ndk/guides/cmake

Reply:
Yeah, can confirm this is a thing I've been doing to get it to compile for a while now.